x-pack/plugins/reporting/export_types/printable_pdf/server/lib/layouts/layout.ts

```diff
@@ -4,7 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import { Size } from '../../../../../types';
-import { ViewZoomWidthHeight } from './types';
+import { ViewZoomWidthHeight } from '../../../../../types';
```

Comment:
`import { Size, ViewZoomWidthHeight } from '../../../../../types';`

Reply:
huh, how the heck did that pass the linter...